readme.md

```diff
@@ -25,7 +25,7 @@ download({
   version: '0.25.1',
   arch: 'ia32',
   platform: 'win32',
-  cache: './zips' // defaults to <user's home directory>/.electron
+  cache: './zips' // defaults to <user's cache directory>/electron-download
```

Comment:
Yes, sharp eye!
This is a leftover from before the 'Cache location' section got added so I removed it as it's duplicate.